lib/active_support/cache/memcached_store.rb

```diff
@@ -80,6 +80,12 @@ def initialize(*addresses, **options)
         extend Strategy::LocalCache
       end
 
+      def append(key, value, options = nil)
+        return true if read_only
+        normalized_key = normalize_key(key, options)
+        @connection.append(normalized_key, value)
```

Comment:
These operation methods also use `handle_exceptions` to provide a fallback and `instrument` to provide an instrumentation hook.

```suggestion
        handle_exceptions(return_value_on_error: nil, on_miss: false) do
          instrument(:append, name) do
            @connection.append(key, value)
          end
          true
        end
```

Which means that you would have to check the return value to see if the append operation succeeded.